enterprise/admin/utils.py

```diff
@@ -112,3 +130,18 @@ def validate_email_to_link(email, raw_email=None, message_template=None, ignore_
             email=email, ec_name=existing_record.enterprise_customer.name
         ))
     return bool(existing_record)
+
+
+def get_course_runs_from_program(program):
+    """
+    Return course runs from program data.
+
```

Comment:
EdX has been asking for arguments listed in the docstrings of new and updated methods - could you add that here and wherever else is relevant?